### PR TITLE
Address Issues Raised in Consensus Call

### PR DESCRIPTION
**Pull request by stpeter**

### CODE REVIEW

File: draft-ietf-uta-rfc6125bis.md [hunk 1]
```diff
@@ -383,15 +383,18 @@ application service type:
...
-: A domain name or host name that is explicitly configured for communicating
-  with the source domain, either by the human user controlling the client
-  or by a trusted administrator.  For example, an IMAP server at mail.example.net
+: A domain name or host name that is explicitly configured at the application layer for communicating
+  with the source domain (e.g., by the human user controlling an application client,
+  by a trusted administrator who sets policy for such clients, or by means of
+  automated configuration in the client itself).  For example, an IMAP server at mail.example.net
   could be a delegated domain for a source domain of example.net associated with an email address of
-  user@example.net.
+  user@example.net.  (This kind of application-layer delegation is not to be confused
```
Comment (richsalz) [inline draft-ietf-uta-rfc6125bis.md:391]: Remove the parens here, this is important, not an "aside"

File: draft-ietf-uta-rfc6125bis.md [hunk 2]
```diff
@@ -480,23 +484,9 @@ that contains additional supplementary information.  Supplementary information
...
-
-2. An "internationalized domain name", i.e., a DNS domain name that includes at
-   least one label containing appropriately encoded Unicode code points
-   outside the traditional US-ASCII range and conforming to the processing
-   and validity checks specified for "IDNA2008" in {{IDNA-DEFS}} and the
-   associated documents. In particular, it contains at least one U-label or
-   A-label, but otherwise may contain any mixture of NR-LDH labels, A-labels,
-   or U-labels.
+In a DNS-ID, and in the DNS domain name portion of an SRV-ID or URI-ID, any
+characters outside the {{US-ASCII}} range are prohibited and internationalized
```
Comment (richsalz) [inline draft-ietf-uta-rfc6125bis.md:488]: I would replace "and" with either a dash or a semi-colon.

File: draft-ietf-uta-rfc6125bis.md [hunk 3]
```diff
@@ -871,8 +861,8 @@ optionally an application service type as follows:
 * A DNS-ID reference identifier MUST be used directly as the DNS domain
   name and there is no application service type.
 
-* An IP-ID reference identifier MUST be exactly equal, octet for octet, to the value of a
-  iPAddress entry in subjectAltName. There is no application service type.
+* An IP-ID reference identifier MUST be exactly equal to the value of a
+  iPAddress entry in subjectAltName, with no partial matching. There is no application service type.
```
Comment (richsalz) [inline draft-ietf-uta-rfc6125bis.md:865]: maybe `no partial (e.g., network-level) matching` ?

File: draft-ietf-uta-rfc6125bis.md [hunk 4]
```diff
@@ -986,9 +981,8 @@ identifiers.
 
 If the identifier is an SRV-ID, then the application service name MUST
 be matched in a case-insensitive manner, in accordance with {{DNS-SRV}}.
-Note that the `_` character is prepended to the service identifier in
-DNS SRV records and in SRV-IDs (per {{SRVNAME}}), and thus does not
-need to be included in any comparison.
+(Note that, per {{SRVNAME}}, the `_` character is part of the service name in
```
Comment (richsalz) [inline draft-ietf-uta-rfc6125bis.md:984]: I'd remove the parens for same reason as above.

File: draft-ietf-uta-rfc6125bis.md [hunk 5]
```diff
@@ -1085,28 +1079,14 @@ in certificate matching.
...
-points that typically are discouraged, either formally or otherwise).
-Although there is little that can be done by certificate matching software
-itself to mitigate these differences (aside from matching exclusively on
-A-labels), the reader needs to be aware that the handling of internationalized
-domain names is inherently complex and can lead to significant security
-vulnerabilities if not properly implemented.
-
-Relevant security considerations for handling of internationalized domain
-names can be found in {{IDNA-DEFS, Section 4.4}}, {{UTS-36}}, and {{UTS-39}}.
+This document specifies only matching between reference identifiers and
```
Comment (richsalz) [inline draft-ietf-uta-rfc6125bis.md:1082]: !!! YEE HA !!!

**Review by richsalz — APPROVED** [review]
minor nits, you can ignore if you want.

**Review by OR13 — APPROVED** [review]

**Comment by richsalz** [comment]
Thanks @OR13 !

**Pull request merged**